### PR TITLE
Lots of good changes to Moon cards, mostly corps.

### PR DESCRIPTION
**Pull request by kberg**
The mechanism for identifying size could very much be simplified, I think.

## Before

image

## After

image

### CODE REVIEW

**Comment by kberg** [comment]
This is running on terraforming-mars-kberg.herokuapp.com

**Comment by kberg** [comment]
This is running on http://terraforming-mars-kberg.herokuapp.com/cards

File: src/components/card/CardCorporationLogo.ts [hunk 1]
```diff
@@ -10,142 +10,157 @@ export const CardCorporationLogo = Vue.component('CardCorporationLogo', {
...
+      case CardName.APHRODITE:
+        return '<div class="card-aphrodite-logo">APHRODITE</div>';
+      case CardName.ARKLIGHT:
+        return '<div class="card-arklight-logo">ARKLIGHT</div>';
+      case CardName.POSEIDON:
+        return '<div class="card-poseidon-logo">POSEIDON</div>';
+      case CardName.SATURN_SYSTEMS:
+        return `<div class="card-saturn-logo">
+        'SATURN <span style="font-size:20px;display:inline-block;">&#x25CF;</span> SYSTEMS';
+        '</span>`;
```
Comment (bafolts) [inline src/components/card/CardCorporationLogo.ts:24]: You didn't introduce this but this should be `</div>` and not `</span>`.

**Review by bafolts — APPROVED** [review]

File: src/components/card/CardCorporationLogo.ts (hunk 1 above)
Comment (kberg) [reply src/components/card/CardCorporationLogo.ts]: Good catch, fixing.

**Pull request merged**